### DISCUSSION TITLE
Cannot Delete Previously-Created Google SSO Account

### DISCUSSION THREAD

**Discussion by brimwats** [tkainrad/keycombiner#77] [Q&A]
Hello! 

continued from #74 

I went to delete the account created by the google SSO in order to merge it with my KC account. When I go to delete account it asks me to login with a password. There wasn't a password made with the SSO account.

**Comment by brimwats** [comment]
alternatively you could feel free to delete the newer account :)

**Comment by tkainrad** [comment]
Hi, 

I remember thinking briefly about this when implementing the account deletion. I thought this should be a rare enough use case that it should be fine if users had to set a password before being able to delete an account. So, to delete the account, please set a password first via _Settings->Password Settings_.

Alternatively, if you want me to delete it, please send an email to support@keycombiner.com using the Gmail account that you signed up with.

**Comment by tkainrad** [comment]
I converted this to be a discussion since it is known behavior and might be more visible as a discussion for people having the same problem. However, I will add some text explaining the situation in the account deletion modal.
